projects/tensorflow/Dockerfile

```diff
@@ -19,8 +19,8 @@ MAINTAINER mihaimaruseac@google.com
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
         curl \
-        python-dev \
-        python-future \
+        python3-dev \
```

Comment:
You shouldn't need this, python 3.8 is in base-builder.

Reply:
Oh, that's better. Remove all python installs